src/dotnet-format.csproj

```diff
@@ -17,6 +17,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 
     <!-- Always run on the latest runtime installed. -->
```

Comment:
I don't know that the comment should say to explain why it needs to be `rc1` but we should do a favor to ourselves and write that down